### PR TITLE
Adding news page for 0.4.0

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Adding news page for 0.4.0. Tagging @travis1111 for awareness.

I need review from the following people for the following areas:
- @elsony for Che
- @sghung for Appsody
- @jopit for Eclipse and VS Code

### CODE REVIEW

**Comment by sishida** [comment]
Questions:
- @sghung In line 15, the last sentence sounds like instructions...do we want to mention that info here or place it in the doc or somewhere else?
- @jopit I saw that you had the links in your original news update, and I kinda like them because, now that we're open source, users can refer to the issues we've been working on. Do you want to include them in the news page, or would you prefer we remove them? I'm fine with either option. :)
Thanks!

**Comment by sghung** [comment]
@sishida I am not sure where "After you update to Codewind 0.4.0, retest all Appsody scenarios because the Appsody init is now performed on the local OS instead of in the Docker container." came from, but that's probably too much information to the user, I think that sentence can be removed.

File: docs/_news/news04.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+#### Che
+- Red Hat OpenStack (RHOS) IBM Cloud Kubernetes Service (IKS) support is available.
+
+#### Eclipse
+- [IDE support is available for configuring template repositories.](https://github.com/eclipse/codewind/issues/32)
+- [HTTPS is supported in the IDEs.](https://github.com/eclipse/codewind/issues/408)
+
+#### VS Code
+- [IDE support is available for configuring template repositories.](https://github.com/eclipse/codewind/issues/32)
```
Comment (jopit) [inline docs/_news/news04.md]: In the IDEs we're using the term `template sources` instead of `template repositories`

File: docs/_news/news04.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#### Appsody
+- Appsody is updated to use the Appsody Codewind Version 0.4.0 extension.
+- When you create an Appsody project, the Codewind installer calls the Appsody extension that is installed as part of the IDE. The Codewind installer does not call the Appsody extension that is installed on the operating system (OS) or the Appsody extension in the Docker container.
+- The Appsody init is no longer called in the Docker container. Instead, the Appsody binary is installed as part of the IDE. With this update, a Docker bind is no longer needed for Appsody creation. After you update to Codewind 0.4.0, retest all Appsody scenarios because the Appsody init is now performed on the local OS instead of in the Docker container.
+
+#### Che
+- Red Hat OpenStack (RHOS) IBM Cloud Kubernetes Service (IKS) support is available.
+
+#### Eclipse
+- [IDE support is available for configuring template repositories.](https://github.com/eclipse/codewind/issues/32)
```
Comment (jopit) [inline docs/_news/news04.md]: In the IDEs we're using the term `template sources` instead of `template repositories`

**Comment by sishida** [comment]
In the latest commit...
@sghung I removed the sentence.
@jopit I have changed "repositories" to "sources."

Thank you!

**Comment by jopit** [comment]
@sishida I think its good to include the links in the news page.

**Pull request merged**